.github/workflows/pull_request.yml

```diff
@@ -1,38 +1,25 @@
-# Copyright 2020 The OpenEBS Authors. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-name: ci
+name: Pull Request CI
 
 on:
   pull_request:
     paths-ignore:
-      - 'deploy/helm/**'
       - 'docs/**'
       - 'design/**'
       - 'changelogs/**'
       - '*.md'
+      - 'MAINTAINERS'
     branches:
-      # on pull requests to develop and release branches
       - develop
-      - 'v*'
+      - 'release/[0-9]+.[0-9]+'
```

Comment:
why not just `release/**` ?

Reply:
That's what I did initially, @niladrih suggested this way.

Reply:
It's like that to narrow it down, it's fine to use `release/*` or `release/**` as well.